docs/index.yaml

```diff
@@ -3,9 +3,9 @@ entries:
   inlets-operator:
```

Comment:
Please revert this change, the maintainers regenerate the chart to prevent the timezones from being changed on every line under `created`

Reply:
Done.